src/main/java/com/example/demo/service/impl/OrderItemsServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.example.demo.service.impl;
+
+
+import com.example.demo.dto.request.OrderItemRequestDto;
+import com.example.demo.dto.response.OrderItemResponseDto;
+import com.example.demo.service.OrderItemsService;
+import java.util.List;
+
+public class OrderItemsServiceImpl implements OrderItemsService {
+    @Override
+    public OrderItemResponseDto createOrderItems(OrderItemRequestDto orderItems) {
+        return null;
+    }
+
+    @Override
+    public List<OrderItemResponseDto> getAllOrderItems() {
+        return null;
+    }
+
+    @Override
+    public OrderItemResponseDto updateOrderItems(int id, OrderItemRequestDto orderItems) {
+        return null;
+    }
+
+    @Override
+    public void deleteOrderItems(int id) {
```

Comment:
```suggestion
    public void deleteOrderItem(int id) {
```
проверьте в других сервисах чтобы было исправлено такое тоже